src/containers/AdminDashboard/AdminController.tsx

```diff
@@ -213,23 +229,22 @@ class AdminController {
   async editPicAPI(evt: { preventDefault: () => void; }): Promise<boolean> {
     evt.preventDefault();
     const { auth, editPic, dispatch } = this.view.props;
-    const {
-      youthName, youthURL, type, showCaption,
+    const { youthName, youthURL, type, showCaption,
     } = this.view.state;
     let r;
     try {
       r = await this.superagent.put(`${process.env.BackendUrl}/book/${editPic._id}`)
-        .set('Authorization', `Bearer ${auth.token}`)
-        .set('Accept', 'application/json')
-        .send({
-          title: youthName, url: youthURL, type, comments: showCaption,
+        .set('Authorization', `Bearer ${auth.token}`).set('Accept', 'application/json')
+        .send({ title: youthName, url: youthURL, type, comments: showCaption,
         });
-    } catch (e) { return Promise.resolve(false); }
+    } catch (e) {
+      this.warnNotif(editPic._id);
```

Comment:
This ID is completely useless, it is the Object ID, you need to pass in meaningful information to the Notification dialog for both Title and Message, this one is for 'Failed to Update the Picture'